app/src/main/java/com/example/myapplication/view/profile/ProfileFragment.kt

```diff
@@ -0,0 +1,70 @@
+package com.example.myapplication.view.profile
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
+import com.example.myapplication.R
+import com.example.myapplication.databinding.FragmentProfileBinding
+
+
+class ProfileFragment : Fragment() {
+    private lateinit var ProfileFragment: FragmentProfileBinding
+    private var _binding: FragmentProfileBinding? = null
+    private val binding get() = _binding!!
+
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentProfileBinding.inflate(layoutInflater, container, false)
+
+        initSettingListener()
+        initTransactionEvent()
+
+        Glide.with(this)
+            .load("https://mblogthumb-phinf.pstatic.net/MjAxOTA0MjNfMjcy/MDAxNTU2MDIwNjg0ODMw.KwUiIDMhdpKzsuNX83GpdFljS1HjgNhCBNcXv2QXfxkg.ksHQVjDUTn8AMV4XVSfETLX-tZ1LTz9-bOmO0o7AtI8g.JPEG.ndh7782/%EC%B9%98%EC%A6%8801.JPG?type=w800")
+            .apply(RequestOptions.circleCropTransform())
+            .into(binding.ivProfile)
+
+        binding.btnFollower.isSelected = true;
+
+        return binding.root
+    }
+
+    fun initTransactionEvent() {
+        val followerFragment = FollowerFragment()
+        val repositstoryFragment = RepositoryFragment()
+
+        getActivity()?.getSupportFragmentManager()
+
+        getActivity()?.supportFragmentManager?.beginTransaction()
+            ?.add(R.id.container_rv, followerFragment)?.commit()
+
+        binding.btnFollower.setOnClickListener {
+            getActivity()?.supportFragmentManager?.beginTransaction()
```

Comment:
```suggestion
            requireActivity()supportFragmentManager.beginTransaction()
```
이렇게 하면 nullable하지 않습니다.

Reply:
많이 늦었지만 알려주신 것대로 코드 모두 수정했습니다! 덕분에 몰랐던 지식을 많이 얻어갑니다 :) 코드리뷰 정말 감사합니다!!!